src/commands/batch.py

```diff
@@ -62,15 +62,15 @@ def list(limit):
             query = session.query(Batch) \
                            .order_by(Batch.created_date.desc()) \
                            .limit(limit)
-            headers = ['ID', 'Date', 'Name', 'Nodes']
+            headers = ['ID', 'Date', 'Command', 'Arguments', 'Nodes']
             rows = []
 
             for batch in query.all():
                 nodes = [job.node for job in batch.jobs]
                 nodes_str = ','.join(nodes)
                 row = [
                     batch.id, batch.created_date, batch.__name__(),
-                    nodes_str
+                    batch.arguments, nodes_str
```

Comment:
Does this still need to be here? I thought `argument` was being removed.

Reply:
In your above comments you asked to remove `Arguments` from `batch history` not `batch list` (you just asked me to change the name for `Command` in list)
As for moving `Exit Code` in `batch history` - I pushed a commit about 2 mins before you posted that, as i was re-reading this & had realised I'd forgotten it :smiley: (the code you've linked even shows the correct order)